miraclelinux/meta-debian-extended
# chrpath: Remove duplicated package

# Purpose of pull request

meta-debian provides chrpath and there is no functionality difference between them.


# Test
## How to test

1. check recipe is removed
2. check functionality

## Test result

meta-debian-extended's recipe is removed.

```
$ bitbake-layers show-recipes | grep -A3 chrpath
chrpath:
  meta-debian          0.16
  meta                 0.16
clutter-1.0:
```
Run chrpath on qemu environment.

```
root@qemuarm64:~# chrpath -l /bin/udevadm 
/bin/udevadm: RPATH=$ORIGIN/src/udev:$ORIGIN/src/shared:$ORIGIN/src/libsystemd:$ORIGIN/src/basic:$ORIGIN/src/libsystemd-network
root@qemuarm64:~# chrpath -d /bin/udevadm 
root@qemuarm64:~# chrpath -l /bin/udevadm 
/bin/udevadm: no rpath or runpath tag found.
```
